server/routes/v1/deployment.ts

```diff
@@ -28,6 +28,17 @@ export const getDeployment = [
   },
 ]
 
+export const getUserDeployments = [
```

Comment:
I think this should be changed to a `getCurrentUserDeployments`, which only returns the current user from `req.user`.  I don't think there's a case where getting an arbitrary persons deployments is needed.

Reply:
Good point, changed.